### ISSUE TITLE
Investigate open-editor package

### ISSUE THREAD

**Issue by joglr** [git-truck/git-truck#602]
This package can be used to open a file in your editor. It can even open to direct column / line numbers.
https://github.com/sindresorhus/open-editor

**Comment by mircealungu** [comment]
nice find. so the package offers a unified way of opening a file in an editor across operating systems? which editor? is it the default on the machine?

**Comment by joglr** [comment]
> nice find. so the package offers a unified way of opening a file in an editor across operating systems? which editor? is it the default on the machine?

Yes, it seems to use the default editor. I have not experienented with it yet.
